apps/desktop/src/lib/gitHost/github/githubPrService.ts

```diff
@@ -20,21 +24,25 @@ export class GitHubPrService implements GitHostPrService {
 		private octokit: Octokit,
 		private repo: RepoInfo,
 		private baseBranch: string,
-		private upstreamName: string,
-		private usePullRequestTemplate?: Persisted<boolean>,
-		private pullRequestTemplatePath?: Persisted<string>
+		private upstreamName: string
 	) {}
 
-	async createPr(title: string, body: string, draft: boolean): Promise<PullRequest> {
+	async createPr({
+		title,
+		body,
+		draft,
+		useTemplate,
+		templatePath
+	}: CreatePullRequestArguments): Promise<PullRequest> {
 		this.loading.set(true);
-		const request = async (pullRequestTemplate: string | undefined = '') => {
+		const request = async (prBody: string | undefined = '') => {
```

Comment:
I'm looking at the default value and I'm wondering why an empty string is better than undefined?

Reply:
Hmm yeah actually the octokit `body` arg can take `undefined` too, so no need for a default here